### PR TITLE
QGB state machine remove invalid validator set check 

### PR DESCRIPTION
**Pull request by SweeXordious**
closes https://github.com/celestiaorg/celestia-app/issues/638

### CODE REVIEW

**Comment by SweeXordious** [comment]
Depends on https://github.com/celestiaorg/celestia-app/pull/645

File: x/qgb/keeper/msg_server.go [hunk 1]
```diff
@@ -47,48 +45,11 @@ func (k msgServer) ValsetConfirm(
 		return nil, sdkerrors.Wrap(types.ErrInvalid, "acc address invalid")
 	}
 
-	// Verify if validator exists
```
Comment (rootulp) [inline x/qgb/keeper/msg_server.go:48]: [question][non-blocking] do any checks need to be removed from https://github.com/celestiaorg/celestia-app/blob/qgb-integration/docs/architecture/ADR-002-QGB-ValSet.md?plain=1#L158 ? I don't see these checks in that ADR section so maybe not but the contents of the ADR don't match 1:1 with the `ValsetConfirm` implementation here so wanted to confirm.

Comment (SweeXordious) [reply x/qgb/keeper/msg_server.go:48]: Yes, the ADRs are pretty oooold now. We will need to update them once we have a stable QGB 1.0.  The reason we stopped updating them, is we were trying a lot of approaches, and we changed the design multiple times. Thus, it was too much work to keep updating them always.

Comment (rootulp) [reply x/qgb/keeper/msg_server.go:48]: Makes sense. If the ADR is no longer accurate then perhaps we can include a disclaimer at the top stating that it's unmaintained and create a GH issue to update it once we have a stable QGB 1.0? Defer to you on how best to track / update it.

Comment (SweeXordious) [reply x/qgb/keeper/msg_server.go:48]: Makes sense. We have few issues concerning this: 
https://github.com/celestiaorg/celestia-app/issues/217
https://github.com/celestiaorg/celestia-app/issues/501
https://github.com/celestiaorg/celestia-app/issues/270

**Review by evan-forbes — APPROVED** [review]
:shipit:

**Pull request merged**